src/componentes/FullModal/FullModal.jsx

```
@@ -1,34 +1,19 @@
-}
-
-export {FullModal}
\ No newline at end of file
+import style from "./FullModal.module.css";
+
+const FullModal = ({ logo, children, back, backLink }) => {
+	return (
+		<div className={style.FullModalContainer}>
+			<div className={style.FullModalLeftContainer}>
+				<a className={style.FullModalBackButton} href={backLink}>
+					<img src={back} />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)